elasticsearchIndexing/process_changed_mods.py

```diff
@@ -22,30 +22,20 @@
 import os
 import shutil
 import sys
-import typing as t
 
 import requests
 
 from elasticsearchIndexing.build_yindex import build_indices
 from elasticsearchIndexing.es_manager import ESManager
 from utility import log
 from utility.create_config import create_config
-from utility.scriptConfig import Arg, BaseScriptConfig
+from utility.script_config_dict import script_config_dict
+from utility.scriptConfig import BaseScriptConfig
 from utility.util import fetch_module_by_schema, validate_revision
 
-
-class ScriptConfig(BaseScriptConfig):
-    def __init__(self):
-        help = 'Process changed modules in a git repo'
-        args: t.List[Arg] = [
-            {
-                'flag': '--config-path',
-                'help': 'Set path to config file',
-                'type': str,
-                'default': os.environ['YANGCATALOG_CONFIG_PATH'],
-            },
-        ]
-        super().__init__(help, args, None if __name__ == '__main__' else [])
+help = script_config_dict['process_changed_mods']['help']
+args = script_config_dict['process_changed_mods']['args']
+DEFAULT_SCRIPT_CONFIG = BaseScriptConfig(help, args, None if __name__ == '__main__' else [])
```

Comment:
This can be refactored into something like:
```python
BASENAME = os.path.basename(__file__)
FILENAME = BASENAME.split('.py')[0]

DEFAULT_SCRIPT_CONFIG = BaseScriptConfig(
        help=script_config_dict[FILENAME]['help'], 
        args=script_config_dict[FILENAME]['args'], 
        None if __name__ == '__main__' else [])
```